_layouts/home.html

```diff
@@ -152,6 +152,28 @@ <h1 class="welcome-header orange-bg">
   ga('create', 'UA-92787920-1', 'auto');
   ga('send', 'pageview');
 
+</script><script>
+
+
+
+var countClicks = (function(){
+
+		var clickAmount = 0;
+
+		return function(){
+
+		  clickAmount += 1;
+
+		  if(clickAmount == 7) {
+
+		     document.getElementsByClassName('home-logo')[0].src = 'assets/fidget spinner.png';
+
+		  }
+
+		}
+
+		})()
```

Comment:
This calls the function immediately, right? not used to (function(){})()
also this should only be counted if you click on the school logo itself, not anywhere, because we want the background game to still be playable